packages/client/e2e/mtv/test-f.spec.ts

```diff
@@ -415,7 +416,8 @@ test('Test F', async ({ browser }) => {
         GEOLOCATION_POSITIONS['Soissons, France'],
     );
     await userBDevice1Page.reload();
-    await initPage(userBDevice1Page);
+    await userBDevice1Page.goto('/');
+    await focusPage(userBDevice1Page);
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** This is a nice refactor!